### PR TITLE
Incorporate Svadu and Svade specs

### LINKED ISSUE #1115: Update A/D bit text and translation algorithm to reflect Svadu extension

### PR DESCRIPTION
**Pull request by aswaterman**
Resolves #1115 

cc @ved-rivos

### CODE REVIEW

File: src/supervisor.adoc [hunk 1]
```diff
@@ -2025,3 +2030,18 @@ HINVAL.GVMA identically to SFENCE.VMA, HFENCE.VVMA, and HFENCE.GVMA,
...
 instructions as no-ops.
 ====
+
+[[sec:svadu]]
+== "Svadu" Standard Extension for Hardware Updating of A/D Bits, Version 1.0
+
+The Svadu extension adds support and CSR controls for hardware updating of PTE A/D bits.
+
+If the Svadu extension is implemented, the `menvcfg`.ADUE field is writable.
+If the hypervisor extension additionally implemented, the `henvcfg`.ADUE field
```
Comment (ved-rivos) [inline src/supervisor.adoc:2040]: is additionally

File: src/supervisor.adoc [hunk 2]
```diff
@@ -2025,3 +2030,18 @@ HINVAL.GVMA identically to SFENCE.VMA, HFENCE.VVMA, and HFENCE.GVMA,
...
+is also writable.
+See <<sec:menvcfg>> and <<sec:henvcfg>> for the definitions of those fields.
+
+<<translation>> defines the semantics hardware updating of A/D bits.  When
```
Comment (ved-rivos) [inline src/supervisor.adoc:2044]: semantics of hardware

File: src/hypervisor.adoc [hunk 3]
```diff
@@ -613,6 +614,15 @@ VS-stage address translation. When PBMTE=0, the implementation behaves
 as though Svpbmt were not implemented for VS-stage address translation.
 If Svpbmt is not implemented, PBMTE is read-only zero.
 
+If the Svadu extension is implemented, the ADUE bit controls whether hardware
+updating of PTE A/D bits is enabled for VS-stage address translation.
+When ADUE=1, hardware updating of PTE A/D bits is enabled during VS-stage
+address translation, and the implementation behaves as though the Svade
+extension were not implemented for VS-mode translation.
```
Comment (ved-rivos) [inline src/hypervisor.adoc:621]: VS-stage address translation.

File: src/machine.adoc [hunk 4]
```diff
@@ -2031,6 +2032,25 @@ with respect to the altered interpretation of page-table entries' PBMT fields.
 See <<hyp-mm-fences>> for additional synchronization requirements when the
 hypervisor extension is implemented.
 
+If the Svadu extension is implemented, the ADUE bit controls whether hardware
+updating of PTE A/D bits is enabled for S-mode and G-stage address
+translations.
+When ADUE=1, hardware updating of PTE A/D bits is enabled during S-mode
+address translation, and the implementation behaves as though the Svade
+extension were not implemented for S-mode translation.
```
Comment (ved-rivos) [inline src/machine.adoc:2040]: S-mode address  translation.

File: src/machine.adoc [hunk 5]
```diff
@@ -2031,6 +2032,25 @@ with respect to the altered interpretation of page-table entries' PBMT fields.
...
+When the hypervisor extension is implemented, if ADUE=1, hardware updating of
+PTE A/D bits is enabled during G-stage address translation, and the
+implementation behaves as though the Svade extension were not implemented for
+G-stage translation.
```
Comment (ved-rivos) [inline src/machine.adoc:2044]: G-stage address translation.

**Review by ved-rivos — APPROVED** [review]
Looks  good to me. Left a few nit comments.

File: src/supervisor.adoc (hunk 1 above)
Comment (aswaterman) [reply src/supervisor.adoc:2040]: Oops, proofreading failure.

**Comment by aswaterman** [comment]
Thanks @ved-rivos!

**Pull request merged**